docs/content/preview/explore/change-data-capture.md

```diff
@@ -166,6 +166,135 @@ table public.projects: INSERT: project_id[integer]:1 name[character varying]:'Pr
 COMMIT 3
 ```
 
+## Try it out with LSN type HYBRID_TIME
+
+### Create a table
+
+Create a table to be streamed using `pg_recvlogical`.
+
+```sql
+CREATE TABLE test (id INT PRIMARY KEY);
+```
+
+### Create a replication slot with LSN type HYBRID_TIME
+
+Create a logical replication slot with the output plugin `test_decoding` and LSN type `HYBRID_TIME` using the following:
+
+```sql
+CREATE_REPLICATION_SLOT test_logical_replication_slot LOGICAL test_decoding HYBRID_TIME;
```

Comment:
To run CREATE_REPLICATION_SLOT sql command a user will need replication connection. If somebody tries to run with a normal connection then this will fail. Better to mention the sql function `pg_create_logical_replication_slot()` here.